Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hydroshare/hs_docker_base:release-1.9.9
+FROM hydroshare/hs_docker_base:3621-django-freshly
```

Comment:
Will the tag be changed to a release version number after the hs_docker_base PR is merged and the new image is built and tagged in dockerhub?

Reply:
yes, once the new hs_docker_base is released, I'll update it.